tests/test-template-tags.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+class Test_Template_Tags_CoAuthors extends CoAuthorsPlus_TestCase {
+
+	/**
+	 * Test get Co-Authors function
+	 */
+	public function test_is_coauthor_for_post() {
+
+		// Test only sending user
+		$this->assertEquals( false, is_coauthor_for_post( '' ) );
+
+		// Test sending user but not post_id
+		$this->assertEquals( false, is_coauthor_for_post( $this->author1 ) );
+
+		// Test sending valid post but no user
+		$this->assertEquals( false, is_coauthor_for_post( '', 1 ) );
+
+		// Check if author1_post1 belongs to author1 as set in coauthorsplus-testcase.php
```

Comment:
What will happen if we run only this file without running `coauthorsplus-testcase.php`?